### PR TITLE
feat: do not show squads and adjust allocations

### PR DESCRIPTION
**Pull request by capJavert**
### Describe what this PR does
- Fix allocations
  - no longer allocates on feeds when post options container renders invisible, it only allocates if menu is open by user
  - no longer allocates on other post types from read button
- No longer show subscribe options for squads since they have their own subscription from squad notification settings

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

File: packages/shared/src/components/fields/PortalMenu.tsx [hunk 1]
```diff
@@ -32,6 +37,7 @@ export interface MenuItemProps<
   label: string;
   action?: (...args: TArgs) => TReturn;
   anchorProps?: TAnchorProps;
+  Wrapper?: ComponentType<{ children: ReactNode }>;
```
[the remark this answers is not in the document]
Comment (capJavert) [reply packages/shared/src/components/fields/PortalMenu.tsx:40]: In action here https://github.com/dailydotdev/apps/pull/2823/files#r1526209880

File: packages/shared/src/components/post/PostHeaderActions.tsx [hunk 2]
```diff
@@ -51,33 +51,38 @@ export function PostHeaderActions({
...
-            target={openNewTab ? '_blank' : '_self'}
-            icon={<OpenLinkIcon />}
-            onClick={onReadArticle}
-            data-testid="postActionsRead"
-          >
-            {!inlineActions && readButtonText}
-          </Button>
-        </SimpleTooltip>
-      )}
+      {!isInternalReadType(post) && !!onReadArticle && <ButtonWithExperiment />}
```
Comment (rebelchris) [inline packages/shared/src/components/post/PostHeaderActions.tsx:85]: This is ok as direct?
Wasn't sure as it looked like you planned to use 2 versions and use withExperiment here?

Main question would be why the change here?

**Review by rebelchris — APPROVED** [review]
Non blocking question

File: packages/shared/src/components/post/PostHeaderActions.tsx (hunk 2 above)
Comment (rebelchris) [reply packages/shared/src/components/post/PostHeaderActions.tsx:85]: Ah got it :)
